.freeipa-pr-ci.yaml

```diff
@@ -1 +1 @@
-ipatests/prci_definitions/gating.yaml
```

Comment:
Please don't mix the temp commit and actual code changes in a single commit.

Reply:
I had amended the previous commit, but the last amend messed things up. Will keep them separate henceforth